addons/io_hubs_addon/third_party/recast.py

```diff
@@ -0,0 +1,636 @@
+        min=0.0,
+        max=30.0,
+        subtype='DISTANCE')
+
+    agent_height: FloatProperty(
+        name="agent_height",
+        description="Agent height",
+        default=AGENT_HEIGHT_DEFAULT,
+        min=0.0,
+        max=30.0,
+        subtype='DISTANCE')
+
+    agent_radius: FloatProperty(
+        name="agent_radius",
+        description="Agent radius",
+        default=AGENT_RADIUS_DEFAULT,
+        min=0.0,
+        max=30.0,
+        subtype='DISTANCE')
+
+    slope_max: FloatProperty(
+        name="slope_max",
+        description="Maximum slope",
+        default=SLOPE_MAX_DEFAULT,
+        min=0.0,
+        max=radians(90),
+        subtype='ANGLE')
+
+    climb_max: FloatProperty(
+        name="climb_max",
+        description="Maximum step height",
+        default=CLIMB_MAX_DEFAULT,
+        min=0.0,
+        max=30.0,
+        subtype='DISTANCE')
+
+    region_min_size: FloatProperty(
+        name="region_min_size",
+        description="Minimum region size",
+        default=REGION_MIN_SIZE,
+        min=0.0,
+        max=30.0,
+        unit='AREA')
+
+    region_merge_size: FloatProperty(
+        name="region_merge_size",
+        description="Merged region size",
+        default=REGION_MERGE_SIZE,
+        min=0.0,
+        max=30.0,
+        unit='AREA')
+
+    edge_max_error: FloatProperty(
+        name="edge_max_error",
+        description="Max edge error",
+        default=EDGE_MAX_ERROR,
+        min=0.0,
+        max=30.0,
+        subtype='DISTANCE')
+
+    edge_max_len: FloatProperty(
+        name="edge_max_len",
+        description="Max edge length",
+        default=EDGE_MAX_LENGTH,
+        min=0.0,
+        max=30.0,
+        subtype='DISTANCE')
+
+    verts_per_poly: IntProperty(
+        name="verts_per_poly",
+        description="Verts per poly",
+        default=VERTS_PER_POLY_DEFAULT,
+        min=3,
+        max=10
+    )
+
+    sample_dist: FloatProperty(
+        name="sample_dist",
+        description="Sample distance",
+        default=SAMPLE_DIST_DEFAULT,
+        min=0.0,
+        max=30.0,
+        subtype='DISTANCE')
+
+    sample_max_error: FloatProperty(
+        name="sample_max_error",
+        description="Max sample error",
+        default=SAMPLE_MAX_ERROR_DEFAULT,
+        min=0.0,
+        max=30.0,
+        subtype='DISTANCE')
+
+    partitioning: EnumProperty(
+        name="partitioning",
+        items=[("WATERSHED", "WATERSHED", "WATERSHED"),
+               ("MONOTONE", "MONOTONE", "MONOTONE"),
+               ("LAYERS", "LAYERS", "LAYERS")],
+        default=PARTITIONING_DEFAULT)
+
+    color: FloatVectorProperty(name="Color",
+                               description="Color",
+                               subtype='COLOR_GAMMA',
+                               default=COLOR_DEFAULT,
+                               size=4,
+                               min=0,
+                               max=1)
+    expanded: BoolProperty(name="expanded", default=True)
```

Comment:
I would have the advanced settings panel default to closed (False), so it doesn't distract from the most commonly changed settings.